### DISCUSSION TITLE
Leave message feature misbehavior

### DISCUSSION THREAD

**Discussion by cngirwa** [LiveHelperChat/livehelperchat#1938] [Q&A]
Hi @all!
When a user leaves a message during offline hours, feedback(Thanks for the message) does not show up to the user, however, the notification does appear in the back office. I upgraded from the old version (v.4.06) to the latest version (v4.21) but the error persists. Is there any setting I'm missing?

**Comment by remdex** [comment]
Here all works
![image](https://user-images.githubusercontent.com/1146085/232434782-4b4617cb-6afd-4172-beb0-6229cf19ca63.png)
